modules/openapi-generator/src/main/resources/swift5/modelEnum.mustache

```diff
@@ -1,7 +1,12 @@
-{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} enum {{classname}}: {{dataType}}, {{#useVapor}}Content, Hashable{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}, CaseIterable {
+{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} enum {{classname}}: {{dataType}}, {{#useVapor}}Content, Hashable{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}, CaseIterable{{^generateFrozenEnums}}{{#vendorExtensions.x-non-frozen-enum-capable}}, CaseIterableDefaultsLast{{/vendorExtensions.x-non-frozen-enum-capable}}{{/generateFrozenEnums}} {
 {{#allowableValues}}
 {{#enumVars}}
     case {{{name}}} = {{{value}}}
 {{/enumVars}}
 {{/allowableValues}}
+{{^generateFrozenEnums}}
+{{#vendorExtensions.x-non-frozen-enum-capable}}
+    case unknown
```

Comment:
@jarrodparkes I'm afraid this will conflict with an Enum case that is already `unknown`, for example, in my project some of the Enum's already have a unknown case.
Here are some alternative names:
- unknownDefault (@unknown default)
- openApiUnknown
- {projectName}Unknown


Do you have other suggestions?

Reply:
@4brunu good call. of your options, I like `unknownDefault` personally

Reply:
@4brunu updated

Reply:
Sorry for didn't put more thought into it, but since I discover that the raw value name is the case name, maybe we should name the enum something a bit more complex to be extra sure that we avoid conflicts.

What do you think of the following suggestions?
- {projectName}UnknownDefault
- unknownDefault{projectName}
- openApiUnknownDefault
- unknownDefaultOpenApi

Do you have other suggestions?

I also like the unknownDefault because of the pattern in Swift @unknown default, but maybe we should try harder to avoid conflicts?

Reply:
Yeh, probably so. I like `unknownDefaultOpenApi`. And now that I think about it, enum cases that are multiword strings, normally use snake_case. With that in mind, I can also set String raw value to `"unknown_default_open_api"`

Reply:
Looks good to me 👍